api/nakadi-event-bus-api.yaml

```diff
@@ -1269,15 +1269,20 @@ definitions:
         description: configuration settings for kafka storage. Only necessary if the storage type is 'kafka'
         type: object
         properties:
+          exhibitor_address:
+            description: the Zookeeper address
+            type: string
+          exhibitor_port:
+            description: the Zookeeper path
+            type: string
           zk_address:
             description: the Zookeeper address
             type: string
           zk_path:
             description: the Zookeeper path
             type: string
         required:
-          - zk_address
-          - zk_path
+          - exhibitor_address
```

Comment:
Explained it offline as a result we want to add description for KafkaConfiguration class.